packages/webview_flutter/webview_flutter_platform_interface/lib/src/platform_webview_controller.dart

```diff
@@ -270,6 +270,13 @@ abstract class PlatformWebViewController extends PlatformInterface {
       'setOnPermissionRequest is not implemented on the current platform',
     );
   }
+
+  /// Toggle the listener for content offset changes. Set to null to stop listening.
+  Future<void> setOnContentOffsetChanged(
```

Comment:
Since we already have `getScrollPosition`, I think I would actually prefer the name `setOnScrollPositionChange` and changing `ContentOffsetChange` -> `ScrollPositionChange`. I totally forgot that we had a precedent for naming scroll events when I initially commented on this..

Reply:
@TheVinhLuong Are you still planning on updating this method name and parameter class name?

Reply:
My bad, I must missed this.